adhoc_customize/CarMsg msg
---
bool success
